metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -39,7 +39,7 @@
 tmp_dir = Script.get_tmp_dir()
 
 hostname = config['hostname']
-metron_group = config['configurations']['cluster-env']['metron_group']
+hadoop_group = config['configurations']['cluster-env']['user_group']
```

Comment:
please put this line in the `hadoop params` section (circa line 80).

Also, why equate hadoop_group to 'user_group'?  In typical hadoop deployments there is already a group named 'hadoop'.  It is managed by Ambari.  In a recent test install of HDP+Metron, it was defined as follows (from /etc/group), which I think would meet your intended use since both storm and metron are included:
* hadoop: x :1004:hive,storm,zookeeper,infra-solr,zeppelin,metron,livy,elasticsearch,spark,flume,kafka,hdfs,yarn,kibana,mapred,hbase,hcat